packages/hoppscotch-cli/src/utils/collections.ts

```diff
@@ -3,16 +3,30 @@ import * as A from "fp-ts/Array";
 import { pipe } from "fp-ts/function";
 import { bold } from "chalk";
 import { log } from "console";
+import { round } from "lodash";
 import { HoppCollection, HoppRESTRequest } from "@hoppscotch/data";
 import { HoppEnvs, CollectionStack, RequestReport } from "../types/request";
-import { preProcessRequest, processRequest } from "./request";
+import {
+  getRequestMetrics,
+  preProcessRequest,
+  processRequest,
+} from "./request";
 import { exceptionColors } from "./getters";
-import { TestReport } from "../interfaces/response";
 import {
   printErrorsReport,
   printFailedTestsReport,
+  printPreRequestMetrics,
+  printRequestsMetrics,
   printTestsMetrics,
 } from "./display";
+import {
+  PreRequestMetrics,
+  RequestMetrics,
+  TestMetrics,
+} from "../types/response";
+import { getTestMetrics } from "./test";
+import { DEFAULT_DURATION_PRECISION } from "./constants";
+import { getPreRequestMetrics } from "./pre-request";
 const { WARN, FAIL } = exceptionColors;
```

Comment:
Add a gap between the imports and the code

Reply:
Done.